apps/web/pages/event-types/index.tsx

```diff
@@ -335,7 +335,7 @@ export const EventTypeList = ({ group, groupIndex, readOnly, types }: EventTypeL
                   <MemoizedItem type={type} group={group} readOnly={readOnly} />
                   <div className="mt-4 hidden sm:mt-0 sm:flex">
                     <div className="flex justify-between space-x-2 rtl:space-x-reverse">
-                      {type.users?.length > 1 && (
+                      {type.team && (
```

Comment:
<img width="1122" alt="Screenshot 2023-02-24 at 12 00 56 AM" src="https://user-images.githubusercontent.com/53316345/220998316-12bc9547-92d0-45df-b784-759102f20d3a.png">

sometimes avatars are not loaded (don't know why but they are present when i do inspect elements).

Everything else works fine  👍

Reply:
Could be something network related 🤔